src/snapshot.rs

```diff
@@ -153,6 +163,30 @@ impl SnapshotCreator {
     }
   }
 
+  pub unsafe fn get_isolate_ptr(&self) -> *mut Isolate {
```

Comment:
Does not need to be unsafe.

Reply:
It looks like this method shouldn't be pub.